test/fixtures/dialects/duckdb/select.sql

```diff
@@ -0,0 +1 @@
+select 10 // 5;
```

Comment:
I figured eventually easier to just keep throwing more stuff into `select.sql` instead of little files for every different little thing.